messages/install.txt

```diff
@@ -0,0 +1,12 @@
+Welcome to Pieces sublime Plugin
+
+
+Please make sure you have the pieces os https://docs.pieces.app/installation-getting-started/what-am-i-installing
```

Comment:
To get started, please make sure you have Pieces OS installed, running, and up-to-date. 

Pieces OS is the centralized storage agent that works on-device, unifying various developer tools (including the Pieces for Developers Desktop Dpp) to proactively capture and enrich useful materials, streamline collaboration, and solve complex problems through a contextual understanding of your unique workflow.

[Get Pieces OS here](https://docs.pieces.app/installation-getting-started/what-am-i-installing)